abci/types/types.proto

```diff
@@ -271,12 +272,18 @@ message PartSetHeader {
 // Validator
 message Validator {
   bytes address = 1;
-  PubKey pub_key = 2 [(gogoproto.nullable)=false];
+  //PubKey pub_key = 2 [(gogoproto.nullable)=false];
   int64 power = 3;
```

Comment:
`3` -> `2`

Reply:
Left this on purpose since it makes it non-breaking and honestly wouldn't be too surprised if pubkeys made their way back !